tools/termscrape.py

```diff
@@ -21,8 +22,8 @@ def scrape_search_results(keyphrase):
         yield example
 
 
-with open("slurprompts.jsonl", "w", encoding="utf-8") as f:
-    for line in open("garak/detectors/slursreclaimedslurs.txt", "r", encoding="utf-8"):
+with open(data_path / "slurprompts.jsonl", "w", encoding="utf-8") as f:
```

Comment:
Since this is writing, perhaps it should be sent to cache and not data. Actually, perhaps it's worth retiring this script from the repo.

Reply:
Glad you noticed something in this file, I needed to flag this file is moved to `tools` and need to validate that `tools` will not be picked up in the `pypi` packaging.

To answer the original question, this tool seemed targeted as a maintainer script that could be used to update the source `data` files.  Since the file should already exist in the repo this external tool `writing` to the file would not violate the `LocalDataPath` expectations that files accessed during `garak` runtime are treated as read only.